integration_tests/test_suites/dagster-gcp-live-tests/gcp_test_proj/defs.py

```diff
@@ -0,0 +1,12 @@
+from dagster import AssetExecutionContext, asset
+
+
+@asset
+def my_asset(context: AssetExecutionContext) -> None:
+    for i in range(10):
+        print(f"Printing without context {i}")  # noqa: T201
+        context.log.info(f"Logging using context {i}")
+    try:
+        raise Exception("This is an exception")
```

Comment:
i am not sure how this is being used / tested.

Reply:
Yea I ought to add a line for it but it shows up in stderr.